assets/scss/base/form/_form.scss

```diff
@@ -227,6 +227,16 @@ Styleguide Form.date
   @include ie-lte(7) {
     zoom: 1;
   }
+
+  input[type="text"] {
```

Comment:
The classes "label--inlineTextinput form-field--inline-text-input" used in the dateFieldsFreeInline helper (https://github.com/hmrc/play-ui/pull/44) were adding a left margin. These classes have been removed so this is no longer needed.